<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rdsOperationWindow</class>
 <widget class="QDialog" name="rdsOperationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Yarra - Data Transfer Client</string>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <property name="whatsThis">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="logGroupbox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>200</height>
      </size>
     </property>
     <property name="title">
      <string>Activity Log</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>7</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>7</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <item>
       <widget class="QTextEdit" name="logEdit">
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
        <property name="undoRedoEnabled">
         <bool>false</bool>
        </property>
        <property name="lineWrapMode">
         <enum>QTextEdit::NoWrap</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="controlGroupbox">
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Control</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <item>
       <widget class="QPushButton" name="transferButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>Transfer Now</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>8</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="hideButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>Close Window</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="postponeButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>Postpone Active Transfer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="logfileButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>Show Log File</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="configurationButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>Configuration</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="shutdownButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>Shut Down Service</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
     <zorder>shutdownButton</zorder>
     <zorder>verticalSpacer</zorder>
     <zorder>postponeButton</zorder>
     <zorder>logfileButton</zorder>
     <zorder>configurationButton</zorder>
     <zorder>verticalSpacer_2</zorder>
     <zorder>transferButton</zorder>
     <zorder>hideButton</zorder>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="infoGroupbox">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Information</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>14</x>
        <y>151</y>
        <width>78</width>
        <height>60</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>78</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>78</width>
        <height>60</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="rds.qrc">:/images/yarralogo.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
      </property>
     </widget>
     <widget class="QLabel" name="InfoLabelSystem">
      <property name="geometry">
       <rect>
        <x>14</x>
        <y>25</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>System Name:</string>
      </property>
     </widget>
     <widget class="QLabel" name="InfoLabelUpdate">
      <property name="geometry">
       <rect>
        <x>14</x>
        <y>43</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Last Update:</string>
      </property>
     </widget>
     <widget class="QLabel" name="InfoValueSystem">
      <property name="geometry">
       <rect>
        <x>95</x>
        <y>25</y>
        <width>351</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Unkown</string>
      </property>
     </widget>
     <widget class="QLabel" name="InfoValueUpdate">
      <property name="geometry">
       <rect>
        <x>95</x>
        <y>43</y>
        <width>351</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Unkown</string>
      </property>
     </widget>
     <widget class="QLabel" name="InfoValueError">
      <property name="geometry">
       <rect>
        <x>14</x>
        <y>72</y>
        <width>421</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>None</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="rds.qrc"/>
 </resources>
 <connections/>
</ui>
